src/Pandell.PackageRunner/Program.cs

```diff
@@ -128,6 +155,7 @@ private int Run(Assembly packageRunnerAssembly, string packageRunnerExeFileName,
             }
 
             // Initializes NuGet repositories
+            Program.WriteLine(verbose, "Initializes NuGet repositories.");
```

Comment:
`Initializing NuGet repositories.` to be consistent with other progress messages.